src/components/SearchMap.js

```diff
@@ -7,7 +7,8 @@ import {MapView} from 'expo';
 const {Marker} = MapView;
 import _ from 'lodash';
 
-// 3rd party libraries - additional
+import SiteCarousel from './SiteCarousel';
```

Comment:
You'll see below a lot of code came out of this `SearchMap` component as I split it into three: `SearchMap`, `SiteCarousel` and `SitePreview`.